modules/SatelliteAgent/SatelliteAgent.hpp

```diff
@@ -80,12 +81,28 @@ class SatelliteAgent : public Everest::ModuleBase {
 
     // ev@211cfdbe-f69a-4cd6-a4ec-f8aaa3d1b6c8:v1
     // insert your private definitions here
-    std::atomic_bool initial_connect{false};
+    bool i_am_here_seen{false};
```

Comment:
Would be nice to have a brief description of the three variables `i_am_here_seen`, `i_am_ready_seen` and `i_am_ready_myself`

Reply:
Yes, I'll add more documentation.